src/flag_gems/ops/isclose.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+from .all import all
+
+
+@pointwise_dynamic(is_tensor=[True, True, False, False], output_dtypes=[torch.bool])
+@triton.jit
+def isclose_func(x, y, rtol, atol):
+    x_fp = x.to(tl.float64)
```

Comment:
maybe for integer & floating point & bool, is_close should use different path?
Is it necesary to cast them all to float64 and then float32?

Reply:
> maybe for integer & floating point & bool, is_close should use different paths(use different JITFunction for different dtypes)? Is it necesary to cast them all to float64 and then float32?

Thanks for the reminder, so I implemente a specific integer JITFunction, because there is no need to judge nan and inf when inputs are integers, so the efficiency will be a little higher. 
Floating point numbers must be converted to float64, otherwise the precision is not enough and it cannot be completely consistent with the judgment result of torch.